python/practices/practice-3/src/path1/symmetrical_numbers.py

```diff
@@ -0,0 +1,48 @@
+"""
+Check and prints the numbers with the last 4 or last 5 digits or all 6 numbers
+that are symmetrical.
+"""
+
+
+import os
+
+
+def reverse_digits(digits):
```

Comment:
Please update the name and docstring, it doesn't match what this function does.

Reply:
Done